tasks/steps/eval_on_control_instance.yml

```diff
@@ -1,36 +1,14 @@
 ---
-- tags:
-    - cartridge-config
-  when: cartridge_eval_body is not none or cartridge_eval_file is not none
-
+- when: cartridge_eval_body is not none or cartridge_eval_file is not none
   block:
     - name: 'BLOCK: Select control instance'
       include_tasks: 'blocks/set_control_instance.yml'
       run_once: true
       when: not control_instance
 
-    - name: 'Set eval body'
-      set_fact:
-        eval_body: >-
-          {{
-            cartridge_eval_body
-            if cartridge_eval_body is not none
-            else lookup("file", cartridge_eval_file)
-          }}
-      run_once: true
+    - import_tasks: 'blocks/eval.yml'
```

Comment:
Let's move `vars` to a place above `import_tasks`

Reply:
I don't like such style.